package.json

```diff
@@ -53,7 +53,9 @@
     "webpack": "^1.7.3"
   },
   "dependencies": {
-    "react": "0.13.1",
+    "classnames": "^1.2.0",
+    "hammerjs": "^2.0.4",
+    "react": "^0.13.1",
```

Comment:
on the one hand: fair enough. On the other: this is literally why we're using semvers: ^ means "any version that isn't a major version upgrade", which is supposed to guarantee stability.